core/src/script/CGXP/widgets/tree/LayerTree.js

```diff
@@ -472,6 +488,19 @@ cgxp.tree.LayerTree = Ext.extend(Ext.tree.TreePanel, {
             action: "delete",
             qtip: this.deleteText
         });
+
+
+        // if there's only one layer in the group, put time slider under the
+        // layer, not directly under group name
+        if (timeSlider) {
+            if (group.children.length == 1) {
+                group.timeSlider = null;
+                group.children[0].timeSlider = timeSlider;
+            } else {
+                group.timeSlider = timeSlider;
+            }
+        }
```

Comment:
It shouldn't be group with only one child but group with only one layer that support time, am I wrong ?